### DISCUSSION TITLE
Episode 165: 2023 Retro! LLMs, Rust vs C++ and More!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#57] [Episode]
Episode Link: https://adspthepodcast.com/2024/01/19/Episode-165.html

Feel free to write comments or ask questions 🙂

**Comment by kentf** [comment]
oO, two seconds of fame! Nice! :D

Conor, if you're heading to either NDC Oslo or NDC TechTown, let it be known on the podcast, and I'll try to aquire tickets to either event and I'll try to make contact!

<sub>A sticker or two would be awesome 😏</sub>

**Reply by codereport** [reply]
NDC Oslo is the one I submitted to. I submitted two talks so hopefully one or both will get in! Will announce on the podcast if it happens.
